packages/data-service-generator/src/models.ts

```diff
@@ -14,6 +14,8 @@ export type Scalars = {
   DateTime: any;
   /** The `JSONObject` scalar type represents JSON objects as specified by [ECMA-404](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf). */
   JSONObject: any;
+  /** The `Upload` scalar type represents a file upload. */
+  Upload: any;
```

Comment:
This is not really good from a TS POV. I'd suggest to configure `codegen` scalar to the appropriate type `GraphQLUpload` 

More info: https://the-guild.dev/graphql/codegen/plugins/typescript/typescript-operations#scalars

Reply:
The resolver already uses a type for the arg, but it is wrapped with an arrow function. can we add the relevant attributes to make sure it is generated correctly in the grpahQL schema?

Reply:
In this case, since the type is managed by the `graphql-upload` dependency  we can leave it as it is.
I've raised a new issue to handle the `gql` types (queries, mutations, fragments) generation for our client to improve our experience and reducing potential issues
- [ ] https://github.com/amplication/amplication/issues/6433